### PR TITLE
Lib32z1 dependency

### PR DESCRIPTION
**Pull request by horviskoadrianko**
I've added lib32z1 because it was missing and mysql database connector deppends on it.

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ RUN apt-get update \
     ca-certificates=20190110 \
     lib32gcc1=1:8.3.0-6 \
     lib32stdc++6=8.3.0-6 \
+    lib32z1 \
```
Comment (timche) [inline base/Dockerfile:8]: Please define the exact version like the other dependencies and install it only on the SourceMod image.

**Review by timche — CHANGES_REQUESTED** [review]
Thanks for fixing this. I've requested changes.

**Review by timche — COMMENTED** [review]
Can you please rebase to latest master? I'd like to see that the image builds, I've added a GitHub workflow.

**Comment by horviskoadrianko** [comment]
I tried to add it to sourcemod Dockerfile but it will throw error unable to locate package. I dont really know how to fix this. I will try tomorrow.

**Comment by timche** [comment]
That's because the [apt cache is removed](https://github.com/timche/docker-csgo/blob/master/base/Dockerfile#L11) in the base image as clean up. You need to run [`apt update`](https://github.com/timche/docker-csgo/blob/master/base/Dockerfile#L3) before you install the dependency.

**Comment by horviskoadrianko** [comment]
Thank you for helping me.

File: sourcemod/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,13 @@
 FROM timche/csgo
 
+USER root
+
+RUN apt-get update \
+    && apt-get install lib32z1=1:1.2.11.dfsg-1 \
```
Comment (timche) [inline sourcemod/Dockerfile:6]:
```suggestion
    && apt-get install -y --no-install-recommends \
    lib32z1=1:1.2.11.dfsg-1 \
```

**Review by timche — CHANGES_REQUESTED** [review]

**Comment by timche** [comment]
Good job 👍 Thanks for fixing this.

**Pull request merged**